bot/codex.py

```
@@ -0,0 +1,483 @@
+        temp_files: list[str] = []
+        for att in turn.attachments:
+            data = att.get("data")
+            if not data:
+                continue
+            path = self._write_attachment(
+                home, data, str(att.get("media_type") or "image/jpeg"))
+            if path:
+                temp_files.append(path)
+                flags += ["-i", path]
+
+        argv = argv + ["exec"]
```

Ruff (0.16.1)

[warning] Consider `[*argv, "exec"]` instead of concatenation

Replace with `[*argv, "exec"]`

(RUF005)